twins/network.go

```diff
@@ -77,10 +95,11 @@ func NewSimpleNetwork(numNodes int) *Network {
 // partitions specifies the network partitions for each view.
 func NewPartitionedNetwork(views []View, dropTypes ...any) *Network {
```

Comment:
May not be related to any changes herein, but I wonder if we could improve the doc comment here. What about?:
```go
// NewPartitionedNetwork creates a new Network with the specified list of views.
// Each view must specify a leader and a set of partitions, each with a set of nodes.
// One or more message types may be specified, which will be dropped at the sending node.
```

Reply:
Very nice. Thanks